utils/node_local_test.py

```diff
@@ -1535,12 +1536,17 @@ def run_query(self, use_json=False, quiet=False):
         print(rc)
         return rc
 
-    def check_usage(self, ino=None, inodes=None, open_files=None, pools=None, containers=None):
+    def check_usage(self, ino=None, inodes=None, open_files=None, pools=None, containers=None,
+                    qpath=None):
         """Query and verify the dfuse statistics.
 
         Returns the raw numbers in a dict.
         """
-        cmd = ['filesystem', 'query', self.dir]
+        cmd = ['filesystem', 'query']
+        if qpath:
+            cmd.append(qpath)
+        else:
+            cmd.append(self.dir)
```

Comment:
Just FYI - no need to change
```suggestion
        cmd = ['filesystem', 'query', qpath or self.dir]
```